test/hotspot/jtreg/runtime/cds/appcds/dynamicArchive/DynamicArchiveRelocationTest.java

```diff
@@ -97,11 +93,7 @@ static void doTest() throws Exception {
 
         // (1) Dump base archive (static)
 
-        OutputAnalyzer out = TestCommon.dumpBaseArchive(baseArchiveName, unlockArg, dumpBaseRelocArg, logArg);
-        if (dump_base_reloc) {
-            out.shouldContain("ArchiveRelocationMode == 1: always allocate class space at an alternative address");
-            out.shouldContain("Relocating archive from");
-        }
+        OutputAnalyzer out = TestCommon.dumpBaseArchive(baseArchiveName, unlockArg, logArg);
```

Comment:
Should the check for `out.shouldContain("Relocating archive from")` be kept?
Otherwise, no need to save the `OutputAnalyzer` returned from `TestCommon.dumpBaseArchive`.

Reply:
I added the `out.shouldContain("Relocating archive from")` back (so that the changes are the same as in ArchiveRelocationTest.java)